fix:Fix a bug where re-editing `filter`and `post filter` input is not possible (#1705)
finos/legend-studio: .changeset/shy-pears-beg.md

---
'@finos/legend-query-builder': patch
---

> Fix a bug where re-editing `filter` and `post filter` input is not possible ([#1594](https://github.com/finos/legend-studio/issues/1594)).

Reviewer: ```suggestion
Fix a bug with the cursor when editing `filter` and `post filter` text input value ([#1594](https://github.com/finos/legend-studio/issues/1594)).
```
